tf_encrypted/tensor/int64.py

```diff
@@ -59,12 +61,12 @@ def native_type(self):
         return tf.int64
 
     def sample_uniform(self, shape: List[int]) -> 'Int64Tensor':
-        value = tf.random_uniform(shape=shape, dtype=self.native_type, minval=tf.int64.min, maxval=tf.int64.max)
+        value = random_func(shape=shape, dtype=self.native_type, minval=tf.int64.min, maxval=tf.int64.max)
```

Comment:
let's stick with the `random_uniform` name! ie something like `tfe.random_uniform`